packages/database/src/migrations/20210421001607-PsssPalauSyndromBubbleMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const syndromes = [
+  {
+    dataElement: 'PSSS_AFR_Daily_Cases',
+    name: 'AFR',
+    customLabel: 'AFR Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_PF_Daily_Cases',
+    name: 'PF',
+    customLabel: 'PF Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_ILI_Daily_Cases',
+    name: 'ILI',
+    customLabel: 'ILI Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_DLI_Daily_Cases',
+    name: 'DLI',
+    customLabel: 'DLI Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_DIA_Daily_Cases',
+    name: 'Diarrhoea',
+    code: 'DIA',
+    customLabel: 'Diarrhoea Case Numbers',
+  },
+  {
+    dataElement: 'PSSS_CON_Daily_Cases',
+    name: 'Conjunctivitis',
+    code: 'CON',
+    customLabel: 'Conjunctivitis Case Numbers',
+  },
+];
+
+const syndromeBaseMapOverlay = {
+  userGroup: 'PSSS Tupaia',
+  isDataRegional: true,
+  dataElementCode: null,
+  measureBuilder: 'sumLatestPerOrgUnit',
+  countryCodes: '{"PW"}',
+  projectCodes: '{"psss"}',
+};
+
+const syndromeRadiusMapOverlay = (SyndromeName, dateElement, SyndromeCode, customLabel) => ({
```

Comment:
Very minor but just for naming consistency:
```suggestion
const syndromeRadiusMapOverlay = (syndromeName, dateElement, syndromeCode, customLabel) => ({
```